connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectFetchErrorDetailsHandler.scala

```diff
@@ -46,9 +44,7 @@ class SparkConnectFetchErrorDetailsHandler(
 
         ErrorUtils.throwableToFetchErrorDetailsResponse(
           st = error,
-          serverStackTraceEnabled = sessionHolder.session.conf.get(
-            Connect.CONNECT_SERVER_STACKTRACE_ENABLED) || sessionHolder.session.conf.get(
```

Comment:
Are we deprecating this config `Connect.CONNECT_SERVER_STACKTRACE_ENABLED`?

Reply:
It's still used, but only verifies the display behavior rather than the stack trace generation.

Reply:
Should we also make Connect.CONNECT_SERVER_STACKTRACE_ENABLED work for Scala client in this pr?

Reply:
This is a bit more weird, in contrast to Python, the server backtrace is always there in Scala, but the user can decide how to print it.